repository-containers/README.md

```diff
@@ -2,9 +2,9 @@
 
 While we encourage you to add development container configuration files to your repositories so others can benefit from them, some organizations prefer to not include development tool related files in source control. To provide additional flexibility, this folder includes dev container definitions that will be be used if you open the cloned repository using [Remote - Containers](https://aka.ms/vscode-remote/download/containers) extension (if there is no `.devcontainer.json` or `.devcontainer/devcontainer.json` file locally). The `images` folder here is used to pre-build images that are then referenced here or directly in source code repositories. 
 
-You may also review [further documentation](https://code.visualstudio.com/docs/remote/create-dev-container#_alternative-repository-configuration-folders) about repository containers.
+You may review [further documentation](https://code.visualstudio.com/docs/remote/create-dev-container#_alternative-repository-configuration-folders) about repository containers to learn how to best configure them.
 
-However, this is **not supported** from [GitHub Codespaces](https://github.com/features/codespaces) currently. 
+This is **not supported** from [GitHub Codespaces](https://github.com/features/codespaces) currently. 
 
 See [CONTRIBUTING.md](../CONTRIBUTING.md) for details on contributing a container.
```

Comment:
Maybe we remove this and say that we are not accepting contributions for new repository containers and that we encourage you to use the Remote - Containers capability to have a local set of them instead?

Reply:
Makes sense. Added:

> We are not accepting contributions for new repository containers. If you are interested in repository containers, we encourage you to use the [Remote - Containers capability](https://code.visualstudio.com/docs/remote/create-dev-container#_alternative-repository-configuration-folders) to have a local set of them.

Reply:
LGTM!